universitat-bremen-institut-fur-politikwissenschaft.csl

```diff
@@ -116,10 +132,15 @@
   </macro>
   <macro name="locator">
     <choose>
-      <if type="article-journal">
-        <text variable="volume" suffix=" "/>
-        <text variable="issue" prefix="(" suffix=")"/>
+      <if type="article-journal" variable="DOI">
+        <text variable="DOI" prefix="DOI: "/>
       </if>
+      <else-if type="article" match="any">
```

Comment:
and as below, this is a misunderstanding of the instructions. The Schimmelfennig article is an example for an E-Journal, not for any journal article with a DOI. You should reverse this: Only where articles don't have a volume number (as is the case for most E-Journals) print the DOI.